functions/go/inflate-helm-chart/go.mod

```diff
@@ -3,8 +3,10 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/inflat
 go 1.16
 
 require (
-	github.com/imdario/mergo v0.3.12
+	github.com/imdario/mergo v0.3.5
 	github.com/pkg/errors v0.9.1
-	sigs.k8s.io/kustomize/api v0.8.11
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	sigs.k8s.io/kustomize/api v0.8.11-0.20210614195535-7e8ba62e9fd9
+	sigs.k8s.io/kustomize/kyaml v0.10.21
```

Comment:
Why not use the latest?

Reply:
changed it to just 0.8.11 (the latest release)